netutils/config/parser.py

```diff
@@ -1375,3 +1375,139 @@ class NetironConfigParser(BaseSpaceConfigParser):
     def banner_end(self) -> str:
         """Demarcate End of Banner char(s)."""
         raise NotImplementedError("Extreme Netiron platform doesn't have a banner.")
+
+
+class FastironConfigParser(BaseSpaceConfigParser):
```

Comment:
Would be curious if subclassing off of `CiscoConfigParser` instead of `BaseSpaceConfigParser` would be better here. Seems like most of the banner logic built there would directly mirror Ruckus implementation.